### PR TITLE
Update URLs

### PR DESCRIPTION
**Pull request by atouchet**

### CODE REVIEW

**Review by Ygg01 — APPROVED** [review]

**Comment by Ygg01** [comment]
Thanks for the fixup. I don't think this warrants a new release, but I could see this going into a new release.

**Comment by Ygg01** [comment]
@atouchet you might want to rebase your branch. I tried resolving conflicts, but it seems the Git doesn't agree with me.

File: Cargo.toml [hunk 1]
```diff
@@ -4,9 +4,10 @@ version = "0.8.0"
 authors = ["Simon Sapin <simon.sapin@exyr.org>"]
 license = "MIT"
 description = "(朽木) HTML/XML tree manipulation library"
-repository = "https://github.com/SimonSapin/kuchiki"
+repository = "https://github.com/kuchiki-rs/kuchiki"
 edition = "2018"
 
+
```
Comment (Ygg01) [inline Cargo.toml:10]: also remove this space, I might have inadvertendly added it when resolving conflicts

**Comment by atouchet** [comment]
It looks like this was done in #88.

**Pull request closed without merging**